src/HotChocolate/Data/src/Data/Filters/Convention/FilterConvention.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using HotChocolate.Types;
+using HotChocolate.Types.Descriptors;
+using HotChocolate.Types.Descriptors.Definitions;
+using System.Collections.Immutable;
+
+namespace HotChocolate.Data.Filters
+{
+    public class FilterConvention :
+        ConventionBase<FilterConventionDefinition>
+        , IFilterConvention
+    {
+        public IImmutableDictionary<int, OperationConvention> Operations { get; private set; }
+
+        private readonly Action<IFilterConventionDescriptor> _configure;
+
+        protected FilterConvention()
+        {
+            _configure = Configure;
+        }
+
+        public FilterConvention(Action<IFilterConventionDescriptor> configure)
+        {
+            _configure = configure
+                ?? throw new ArgumentNullException(nameof(configure));
+        }
+
+        protected override FilterConventionDefinition CreateDefinition(
+            IConventionContext context)
+        {
+            var descriptor = FilterConventionDescriptor.New(context);
+            _configure(descriptor);
+            return descriptor.CreateDefinition();
+        }
+
+        protected virtual void Configure(
+            IFilterConventionDescriptor descriptor)
+        {
+        }
+
+        protected override void OnComplete(
+            IConventionContext context,
+            FilterConventionDefinition? definition)
+        {
+            Operations = definition
+                .Operations
+                .ToImmutableDictionary(x => x.Operation, x => new OperationConvention(x));
+        }
+
+        public NameString GetFieldDescription(IDescriptorContext context, MemberInfo member)
+            => context.Naming.GetMemberDescription(member, MemberKind.InputObjectField);
```

Comment:
we have to find one style on this.

A)

```csharp
public NameString GetFieldDescription(IDescriptorContext context, MemberInfo member)
    => context.Naming.GetMemberDescription(member, MemberKind.InputObjectField);
````

B)

```csharp
public NameString GetFieldDescription(IDescriptorContext context, MemberInfo member) => 
    context.Naming.GetMemberDescription(member, MemberKind.InputObjectField);
````

@rstaib @PascalSenn what is each once preference?

Reply:
B)

Reply:
B)